modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
```

Comment:
So, I've read the entire document. Reading the title, I thought I'd be forwarding my current, existing logs to Amazon CloudWatch. In this document, I have information and instructions to:

- replace prefixes in log group names
- naming log groups for application namespaces 
- naming log groups for UUIDs

So I'm wondering if the title best captures all the great content and information you detail here in this document. Specifically, I'm not sure if the verb, `forwarding` is the best action that entails all the actions users complete in this doc :)